docs/templates/index.md

```diff
@@ -69,14 +69,37 @@ coder templates create <template-name>
 > [examples/](https://github.com/coder/coder/tree/main/examples/templates)
 > directory in the repo.
 
-## Configure Max Workspace Autostop
+## Template policies (enterprise)
 
-To control cost, specify a maximum time to live flag for a template in hours or
-minutes.
+By default, developers have unlimited workspace usage and can use any template version. Template policies allows you to:
 
-```sh
-coder templates create my-template --default-ttl 4h
-```
+- Auto-stop workspaces after 3 hours of inactivity.
+- Require workspaces to be restarted frequently to ensure they use the most recent template version.
+- Disable workspace auto-start.
+
+### Stop policy
+
+Stop policies require workspaces be stopped (or restarted), regardless of activity:
+
+- Daily (Restart on next quiet hours)
+- Weekly (Restart on weekends)
+- Custom (Every 2 weeks)
+
+  ```text
+   Every [2] [Weeks] on [Friday]
+   or
+   Every [2] [Days]
+   or
+   Every [4] [Weeks] on [Saturday]
+   ```
+
+- Never
+
+Stops occur during a user's [quiet hours](../workspaces.md#quiet-hours), which prevent interruptions. If a user is still using their workspace, they can postpone a restart up to 3 hours.
```

Comment:
What if a user doesn't have their quiet hours enabled? I think the policy should to work regardless to provide a strong guarantee.

I also think we should declare somewhere in the docs that we intend on providing a strong guarantee of policy enforcement.

Reply:
Allowing users to post-pone a policy stop seems to undermine a strong guarantee as well

Reply:
I was thinking:

- By default, all users have quiet hours of 12-4 AM based on the server timezone
- Users can begin postponing quiet hours at 12 AM
- Since quiet hours can only be bumped 3x, a shut down is guaranteed by 4 AM. 

Happy to change docs to clarify if that all sounds good

Reply:
"server timezone" needs some definition, because when there are multiple coder daemons they may differ.

👍🏽 to adding a limit of post-pone

So, when an auto-stop policy is enabled on a template, users can no longer disable quiet hours? It would be good to say that explicitely.

Reply:
My implementation was going to require the operator to set a default quiet hours with timezone before the feature could be enabled

Reply:
Each user must have quiet hours because it's a site-wide setting per user, either they have their own custom quiet hours or they use the default. If a template doesn't have a max workspace lifetime then it just won't apply to workspaces on that template

Reply:
What would be the case when developers are distributed around different TZ? 12-4 AM in PST TZ isn't an excellent quiet hours window for someone in Sydney.